src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.spec.ts

```diff
@@ -459,4 +459,28 @@ describe('ReIssueCertificateComponent', () => {
     expect(component.unsubscribe$.next).toHaveBeenCalled();
     expect(component.unsubscribe$.complete).toHaveBeenCalled();
   });
+
+  it ('should return TRUE when the status is 2', () => {
+    const batch = {batchId: '1', name: 'batch 1', certificates: [], ssuedCertificates: [], createdBy: '123', status: 2 };
+    const enableData = component.enableReIssueCert(batch);
+    expect(enableData).toBeTruthy();
+  });
+
+  it ('should return TRUE when the certificates are issued', () => {
```

Comment:
Update statement easy to understand.
'should enable "re-issue" button, when pdf certificate is issued("certificates" exisit)'